NEWS.md

```diff
@@ -0,0 +1,10 @@
+SelfControlledCohort 1.6.0
+==========================
+- Added changelog in NEWS.md
+- Added support for github actions and removed travis
+- Added computeTarDistribution flag, which computes the distribution of exposure time
+ windows for subjects experiencing the outcome and average absolute time between
+ exposure and outcome.
+ - Added guide for usage with some examples
```

Comment:
I think you're referring to the vignette? I would use the word 'vignette', which should be well know to R users.